### PR TITLE
Updates to ergo ecosystem

### PR DESCRIPTION
**Pull request by glasgowm148**

### CODE REVIEW

**Comment by glasgowm148** [comment]
Could this be merged please 🙏 @eherrerosj @jubos

File: data/ecosystems/c/cardano.toml [hunk 1]
```diff
@@ -24,7 +23,7 @@ sub_ecosystems = [
   "Seal Finance",
   "SingularityNET",
   "SpaceBudz",
-  "Spectrum Labs",
+  "spectrum-labs",
```
Comment (eherrerosj) [inline data/ecosystems/c/cardano.toml:26]: Can you keep the original name please? We prefer human readable ecosystem names

File: data/ecosystems/e/ergo-auction-house.toml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Ecosystem Level Information
-title = "Ergo Auction House"
+title = "ergo-auction-house"
```
Comment (eherrerosj) [inline data/ecosystems/e/ergo-auction-house.toml:2]: Can you keep the original name please? We prefer human readable ecosystem names

File: data/ecosystems/e/ergo-education.toml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # Ecosystem Level Information
-title = "Ergo Education"
+title = "ergo-education"
```
Comment (eherrerosj) [inline data/ecosystems/e/ergo-education.toml:2]: Can you keep the original name please? We prefer human readable ecosystem names

File: data/ecosystems/e/ergo-hackathons.toml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 # Ecosystem Level Information
-title = "Ergo Hackathons"
+title = "ergo-hackathons"
```
Comment (eherrerosj) [inline data/ecosystems/e/ergo-hackathons.toml:2]: Can you keep the original name please? We prefer human readable ecosystem names

File: data/ecosystems/e/ergo-miner-tooling.toml [hunk 5]
```diff
@@ -1,5 +1,5 @@
 # Ecosystem Level Information
-title = "Ergo Miner Tooling"
+title = "ergo-miner-tooling"
```
Comment (eherrerosj) [inline data/ecosystems/e/ergo-miner-tooling.toml:2]: Can you keep the original name please? We prefer human readable ecosystem names

File: data/ecosystems/e/ergo-labs.toml [hunk 6]
```diff
@@ -1,91 +0,0 @@
-# Ecosystem Level Information
```
Comment (eherrerosj) [inline data/ecosystems/e/ergo-labs.toml:1]: Why is this file removed?

File: data/ecosystems/e/ergo-names.toml [hunk 7]
```diff
@@ -1,5 +1,5 @@
 # Ecosystem Level Information
-title = "Ergo Names"
+title = "ergo-names"
```
Comment (eherrerosj) [inline data/ecosystems/e/ergo-names.toml:2]: Can you keep the original name please? We prefer human readable ecosystem names

File: data/ecosystems/e/ergo-raffle.toml [hunk 8]
```diff
@@ -1,5 +1,5 @@
 # Ecosystem Level Information
-title = "Ergo Raffle"
+title = "ergo-raffle"
```
Comment (eherrerosj) [inline data/ecosystems/e/ergo-raffle.toml:2]: Can you keep the original name please? We prefer human readable ecosystem names

File: data/ecosystems/e/ergo-stats-analysis.toml [hunk 9]
```diff
@@ -1,5 +1,5 @@
 # Ecosystem Level Information
-title = "Ergo Stats & Analysis"
+title = "ergo-stats-analysis"
```
Comment (eherrerosj) [inline data/ecosystems/e/ergo-stats-analysis.toml:2]: Can you keep the original name please? We prefer human readable ecosystem names

File: data/ecosystems/s/sigmausd.toml [hunk 10]
```diff
@@ -9,8 +9,10 @@ github_organizations = []
 [[repo]]
 url = "https://github.com/anon-real/sigma-usd"
 
-[[repo]]
-url = "https://github.com/bdkent/sigmausd-history"
+
+#[[repo]] excluding as it generates a ton of commits
```
Comment (eherrerosj) [inline data/ecosystems/s/sigmausd.toml:13]: nice catch!

File: data/ecosystems/r/rosen-bridge.toml [hunk 11]
```diff
@@ -0,0 +1,64 @@
+# Ecosystem Level Information
+title = "rosen-bridge"
```
Comment (eherrerosj) [inline data/ecosystems/r/rosen-bridge.toml:2]: Can you keep the original name please? We prefer human readable ecosystem names

File: data/ecosystems/n/nightowl-casino.toml [hunk 12]
```diff
@@ -1,5 +1,5 @@
 # Ecosystem Level Information
-title = "NightOwl Casino"
+title = "nightowl-casino"
```
Comment (eherrerosj) [inline data/ecosystems/n/nightowl-casino.toml:2]: Can you keep the original name please? We prefer human readable ecosystem names

File: data/ecosystems/l/lithos-protocol.toml [hunk 13]
```diff
@@ -1,5 +1,5 @@
 # Ecosystem Level Information
-title = "Lithos Protocol"
+title = "lithos-protocol"
```
Comment (eherrerosj) [inline data/ecosystems/l/lithos-protocol.toml:2]: Can you keep the original name please? We prefer human readable ecosystem names

File: data/ecosystems/g/grand-gambit.toml [hunk 14]
```diff
@@ -1,5 +1,5 @@
 # Ecosystem Level Information
-title = "Grand Gambit"
+title = "grand-gambit"
```
Comment (eherrerosj) [inline data/ecosystems/g/grand-gambit.toml:2]: Can you keep the original name please? We prefer human readable ecosystem names

File: data/ecosystems/e/ergo.toml [hunk 15]
```diff
@@ -235,7 +263,16 @@ url = "https://github.com/ergoplatform/static-data"
 url = "https://github.com/ergoplatform/website"
 
 [[repo]]
-url = "https://github.com/mgpai22/ergosapiens"
```
Comment (eherrerosj) [inline data/ecosystems/e/ergo.toml:238]: why was this one removed?

**Review by eherrerosj — CHANGES_REQUESTED** [review]
Nice PR! Thanks for the contribution. I left some comments (most of them regarding renaming to non-human readable format). Let's workout those details and we should be good to merge

File: data/ecosystems/s/swamp-audio.toml [hunk 16]
```diff
@@ -1,5 +1,5 @@
 # Ecosystem Level Information
-title = "Swamp Audio"
+title = "swamp-audio"
```
Comment (eherrerosj) [inline data/ecosystems/s/swamp-audio.toml:2]: Can you keep the original name please? We prefer human readable ecosystem names

File: data/ecosystems/z/zengate-global.toml [hunk 17]
```diff
@@ -1,5 +1,5 @@
 # Ecosystem Level Information
-title = "ZenGate Global"
+title = "zengate-global"
```
Comment (eherrerosj) [inline data/ecosystems/z/zengate-global.toml:2]: Can you keep the original name please? We prefer human readable ecosystem names

File: data/ecosystems/z/zk-treasury.toml [hunk 18]
```diff
@@ -1,5 +1,5 @@
 # Ecosystem Level Information
-title = "ZK Treasury"
+title = "zk-treasury"
```
Comment (eherrerosj) [inline data/ecosystems/z/zk-treasury.toml:2]: Can you keep the original name please? We prefer human readable ecosystem names

**Review by eherrerosj — CHANGES_REQUESTED** [review]
forgot to comment on a few ecosystem names to keep the original human-readable ones

**Comment by glasgowm148** [comment]
Updated

File: data/ecosystems/z/zengate-global.toml [hunk 19]
```diff
@@ -1,5 +1,5 @@
 # Ecosystem Level Information
-title = "ZenGate Global"
+title = "Zengate Global"
```
Comment (eherrerosj) [inline data/ecosystems/z/zengate-global.toml:2]: please keep the original name as is

File: data/ecosystems/e/ergo.toml [hunk 20]
```diff
@@ -2,50 +2,78 @@
...
+    # NFTs 
+    "SkyHarbor",
+    "Ergo Auction House",
+
+    # Dev
+    "FleetSDK",
+    "Kiosk",
+    "Azorus",
+    "Ergo Education", 
+    "ergo-developer-tooling",
```
Comment (eherrerosj) [inline data/ecosystems/e/ergo.toml:54]: please keep the original readable name

File: data/ecosystems/e/ergo-labs.toml (hunk 6 above)
Comment (eherrerosj) [reply data/ecosystems/e/ergo-labs.toml:1]: @glasgowm148

**Review by eherrerosj — CHANGES_REQUESTED** [review]
Almost there, left a couples comments and questions

**Comment by glasgowm148** [comment]
Thanks for your help!

**Pull request merged**